src/content/url-handling.js

```diff
@@ -97,8 +101,9 @@ export function stateFromCurrentLocation() {
         selectedThread: 0,
         callTreeSearchString: '',
         callTreeFilters: {},
-        jsOnly: false,
+        implementation: 'all',
         invertCallstack: false,
+        hidePlatformDetails: false,
```

Comment:
I'm not quite sure what you mean here, the initial state is defined in the reducer unless I'm missing something. I think having flow here now will give us a bit more safety on the url handling being correct.